dags/legacy_test/tests/pytorch/r2.3/common.libsonnet

```diff
@@ -0,0 +1,215 @@
+// Copyright 2024 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+local common = import '../common.libsonnet';
+local experimental = import '../experimental.libsonnet';
+local mixins = import 'templates/mixins.libsonnet';
+local utils = import 'templates/utils.libsonnet';
+local volumes = import 'templates/volumes.libsonnet';
+
+{
+  local r2_3 = {
+    frameworkPrefix: 'pt-2_3',
+    tpuSettings+: {
+      softwareVersion: 'tpu-ubuntu2204-base',
+    },
+    imageTag: 'r2.3.0-rc2_3.10',
```

Comment:
Are we updating from 3.8 to 3.10 build of pytorch?

Reply:
Hmm, just noticed that in nightly, the PT is specified using 3.8 while the torch_xla is 3.10 build. You are good here!